recipe/meta.yaml

```diff
@@ -24,78 +24,80 @@ requirements:
     - python-build
     - meson-python ==0.13.1
     - meson ==1.2.1
-    - cython >=0.29.33,<3
-    - numpy {{ numpy }}
+    - cython >=3.0.5
+    - numpy 1.22  # [py<311]
+    - numpy {{ numpy }}  # [py>=311]
     - versioneer
-    - tomli # [py<311]
     - pip
   run:
     - python
     # Base Dependencies
     - {{ pin_compatible('numpy') }}
     - python-dateutil >=2.8.2
     - pytz >=2020.1
-    - python-tzdata >=2022.1
+    - python-tzdata >=2022.7
     # Recommended dependencies to achieve large speedups
     # see https://pandas.pydata.org/pandas-docs/version/2.1.1/getting_started/install.html#performance-dependencies-recommended
-    - numexpr >=2.8.0
-    - bottleneck >=1.3.4
+    - numexpr >=2.8.4
+    - bottleneck >=1.3.6
   run_constrained:
     # Optional dependencies
     # see https://pandas.pydata.org/pandas-docs/version/2.1.1/getting_started/install.html#optional-dependencies
     # and https://github.com/pandas-dev/pandas/blob/v2.1.1/pyproject.toml
     # performance
-    - numba >=0.55.2
+    - numba >=0.56.4
```

Comment:
how about moving numba into `run`? Also highlighted as a performance-enhancing package

Reply:
We build pandas on all platforms while numba isn't compatible with all platforms we support (for example s390x). So maybe that's why we don't add it into the runtime requirements.

Reply:
Yeah I had the same thought as you Dan, and JC's comment is right. See the build that failed here https://github.com/AnacondaRecipes/pandas-feedstock/pull/43/commits/24909f57bece4e0298836d39771ba6b55f8e0aad